test/wasm2js/target_js.wast

```diff
@@ -0,0 +1,29 @@
+ (func $0 (param $0 i32) (result i32)
```

Comment:
Please make the testcase minimal - we don't need all the entries in the table here, I am pretty sure (do we need the elem at all?)

Reply:
@kripken Many thanks for reviewing this. Shrinking the test case any smaller than the following seems to result, as best as I can tell, with `wasm2js -O1` failing to keep enough code in to hit unreachable in the before case. To maintain the unreachable before/after problem, I was able to shrink the table, simplify the elem, and remove the superfluous first function. It's possible I'm missing additional minimization but am not sure how to rework further.

```
(module
 (type $i32_=>_none (func (param i32)))
 (import "env" "memory" (memory $mimport$0 256 32768))
 (table $0 3 3 funcref)
 (elem (i32.const 1) $0)
 (func $0 (param $0 i32) (param $1 i32)
  (i32.store
   (i32.const 1)
   (i32.xor
    (i32.shl
     (i32.const 1)
     (local.get $0)
    )
    (i32.const -1)
   )
  )
  (call_indirect (type $i32_=>_none)
   (i32.const 1)
   (i32.const 0)
  )
  (unreachable)
 )
)
```